portal-web/build.xml

```diff
@@ -106,6 +106,7 @@
 			outputproperty="jspc.output"
 		>
 			<jvmarg if:true="${jvm.debug}" line="${jpda.settings}" />
+			<sysproperty key="log4j2.disable.jmx" value="true" />
```

Comment:
@yuhai @dantewang Please add this property to system.properties, it should also work.

Reply:
@tinatian, got it. worked.